### ISSUE TITLE
Generate init_*.lua files

### ISSUE THREAD

**Issue by Sheeo** [FAForever/client#345]
Rather than having .init.lua files sent statically by the server as they are right now, the client should genrate them based on the `mod_info.lua` file.

The `mod_info.lua` spec has already been extended to support this.

**Acceptance Criteria**
- Given the example featured mod 'FAF', a lua with with the same global `path` variable as executing the current `init_faf.lua` file should be generated.

**Comment by Wesmania** [comment]
Is this still relevant?

**Comment by micheljung** [comment]
yes, see [a reference implementation here](https://github.com/FAForever/downlords-faf-client/blob/129ba0461294f78407c6d6276062d96dc88ef865/src/main/java/com/faforever/client/game/FaInitGenerator.java)
along with [a template init.lua](https://github.com/FAForever/downlords-faf-client/blob/129ba0461294f78407c6d6276062d96dc88ef865/src/main/resources/fa/init_template.lua)
